### PR TITLE
Lb gnome mercy module html

### PR DESCRIPTION
**Pull request by LucasBress0n**
Still missing current functionality for each event listener, Awaiting Transient State
Forgor to add Submit.JS one secondo

### CODE REVIEW

**Review by Lincoln-WIlcox — APPROVED** [review]

File: src/scripts/GnomeMercy.js [hunk 1]
```diff
@@ -5,24 +5,24 @@
...
+  const CraftFormHTML = await CraftForm();
+  const CompletionsHTML = await Completions();
+  const requestFormHTML = await RequestForm();
+
   return `
     <h1>Gnome Mercy</h1>
-    <article id="request">
-      
-    </article>
+    ${requestFormHTML}
```
Comment (ztrouy) [inline src/scripts/GnomeMercy.js:19]: Could you add an article tag around the Request Form to match the Craft Form and Completions sections? The class and id can match what is in the other two

**Review by TheAdameia — APPROVED** [review]

File: src/scripts/CompleteButton.js [hunk 2]
```diff
@@ -19,16 +17,12 @@ const onSubmitRequestPressed = (event) =>
...
-{
-  document.addEventListener(
-    "mouseup",
-    onSubmitRequestPressed
-  )
+export const CompleteButton = () => {
+  document.addEventListener("mouseup", onSubmitRequestPressed);
 
-  let returnHTML = `<button class="complete-button__button" id="submit-request">Submit Request</button>`
+  let returnHTML = `<button class="complete-button__button formSection" id="submit-request">Submit Request</button>`;
```
Comment (ztrouy) [inline src/scripts/CompleteButton.js:25]: The added formSection class is messing with formatting

File: src/scripts/PurposeFormField.js [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-    console.log(event.target.value);
+    setCraftIntentChoice(event.target.value);
   }
 });
 
 export const PurposeFormField = () => {
   return `
-    <label class="label" for="purpose">Purpose</label>
-    <input placeholder="What should the effect be?" type="text" data-id="purpose" class="input">
+    <label class="label formSection" for="purpose">Purpose</label>
```
Comment (ztrouy) [inline src/scripts/PurposeFormField.js:11]: The added formSection class is messing with formatting

File: src/scripts/PurposeFormField.js [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
+    <input placeholder="What should the effect be?" type="text" data-id="purpose" class="formSection"/>
```
Comment (ztrouy) [inline src/scripts/PurposeFormField.js:12]: The added formSection class is messing with formatting

File: src/scripts/TypeField.js [hunk 5]
```diff
@@ -11,8 +11,8 @@ export const TypeField = async () => {
     res.json()
   );
   let html = `
-  <label class="label" for="type">Type</label>
-    <select data-id="TypeFieldForm">
+  <label class="formSection" for="type">Type</label>
```
Comment (ztrouy) [inline src/scripts/TypeField.js:14]: The added formSection class is messing with formatting

File: src/scripts/TypeField.js [hunk 6]
```diff
@@ -11,8 +11,8 @@ export const TypeField = async () => {
...
+    <select data-id="TypeFieldForm" class="formSection">
```
Comment (ztrouy) [inline src/scripts/TypeField.js:15]: The added formSection class is messing with formatting

**Review by ztrouy — CHANGES_REQUESTED** [review]
The formSection class added to `NameFormField.js`, `PurposeFormField.js`, `TypeField.js`, and `CompleteButton.js` mess with the formatting in a weird way. Please revert those changes. They appear to be fine to use for the articles in `GnomeMercy.js` though

File: src/scripts/CompleteButton.js
[the remark this answers is not in the document]
Comment (LucasBress0n) [reply src/scripts/CompleteButton.js:1]: Nope, Whenever saving a JS file, my VSCode, Prettier extension automatically reformates things, mostly by adding ; to signify a specific line of code closing.

File: src/scripts/GnomeMercy.js (hunk 1 above)
Comment (LucasBress0n) [reply src/scripts/GnomeMercy.js]: Done

File: src/scripts/CompleteButton.js (hunk 2 above)
Comment (LucasBress0n) [reply src/scripts/CompleteButton.js]: Prettier Extension.

**Review by ztrouy — APPROVED** [review]

**Pull request merged**